optimum/onnxruntime/configuration.py

```diff
@@ -604,73 +608,222 @@ def tensorrt(
 class OptimizationConfig:
     """
     OptimizationConfig is the configuration class handling all the ONNX Runtime optimization parameters.
+    There are two stacks of optimizations:
+        1. The ONNX Runtime general-purpose optimization tool: it can work on any ONNX model.
+        2. The ONNX Runtime transformers optimization tool: it can only work on a subset of transformers models.
 
-    Args:
-        optimization_level (`int`, defaults to 1):
-            ONNX opset version to export the model with.
+    Attributes:
+        general_optimization_level (`int`, defaults to 1):
             Optimization level performed by ONNX Runtime of the loaded graph.
             Supported optimization level are 0, 1, 2 and 99.
-            0 will disable all optimizations.
-            1 will enable basic optimizations.
-            2 will enable basic and extended optimizations, including complex node fusions applied to the nodes
-            assigned to the CPU or CUDA execution provider, making the resulting optimized graph hardware dependent.
-            99 will enable all available optimizations including layout optimizations.
+                - 0: will disable all optimizations
+                - 1: will enable basic optimizations
+                - 2: will enable basic and extended optimizations, including complex node fusions applied to the nodes
+                assigned to the CPU or CUDA execution provider, making the resulting optimized graph hardware dependent
+                - 99: will enable all available optimizations including layout optimizations
         optimize_for_gpu (`bool`, defaults to `False`):
             Whether to optimize the model for GPU inference.
             The optimized graph might contain operators for GPU or CPU only when `optimization_level` > 1.
         fp16 (`bool`, defaults to `False`):
             Whether all weights and nodes should be converted from float32 to float16.
-        optimize_with_onnxruntime_only (`bool`, defaults to `False`):
-            Whether to only use ONNX Runtime to optimize the model and no graph fusion in Python.
-        disable_gelu (`bool`, defaults to `False`):
+        enable_transformers_specific_optimizations (`bool`, defaults to `True`):
```

Comment:
I went for `enable_transformers_specific_optimizations`, which I think is nice because it conveys that we will use a `transformers` specific tool.
 @echarlaix @regisss (did not use `fuse_operators` since you also have approximations such as fast gelu)

Reply:
Sounds good to me :+1: